app/models/cart.rb

```diff
@@ -0,0 +1,4 @@
+class Cart < ActiveRecord::Base
+	belongs_to :user
+	has_and_belongs_to_many :product_variants
```

Comment:
The join model will need to have a quantity field because one user can buy for example 3 pairs of the same shoe. So, I suggest making a new model, instead of using has_and_belongs_to_many